Scripts/Mend SCA/README.md

```diff
@@ -273,7 +273,37 @@ The [cache-ua.sh](cache-ua.sh) script can be added to the CI/CD pipeline on a st
 
 ```
 curl -LJO https://raw.githubusercontent.com/mend-toolkit/mend-examples/main/Scripts/cache-ua.sh.sh 
-chmod +x ./cache-ua.sh.sh && ./cache-ua.sh.sh
+chmod +x ./cache-ua.sh && ./cache-ua.sh
 ```
 
 See additional example for implementation within a build pipeline under [CI-CD](../CI-CD/README.md#caching-the-unified-agent) (`*-cached-ua.yml`).  
+
+<br>
+<hr>
+
+## Pending task cleanup
+
+[pending-task-cleanup.sh](pending-task-cleanup.sh)  
+
+This script allows the user to cleanup outstanding [Pending Tasks](https://docs.mend.io/bundle/wsk/page/ui_-_request_history_report_and_pending_tasks.html) in the Mend SCA UI, when this process is no longer required. Please ensure the setting '[Open pending tasks for new libraries](https://docs.mend.io/bundle/wsk/page/ui_-_request_history_report_and_pending_tasks.html)' is disabled under the Integrate, Advanced Settings area. In addition, also ensure that their are no [Policies](https://docs.mend.io/bundle/sca_user_guide/page/managing_automated_policies.html#Applying-Actions-to-a-Library) that are 'Reassign' or 'Condition', which could create new tasks. 
+
+The [pending-task-cleanup.sh](pending-task-cleanup.sh) script is designed to be executed one time per organisation to clean up historic pending requests the Mend SCA UI. 
+
+<br>
+
+**Prerequisites:**  
+
+* `jq` and `curl` must be installed
+* The tasks within Mend should be assigned to a user and not to a group (Edit policy->Reasssign->Assign to User)
+
+<br>
+
+**Execution:**  
+
+```
+export MEND_URL=example-https://saas.mend.io
```

Comment:
remove example- and just leave the example

Reply:
done